### PR TITLE
ADEN-5694 implement scrollY tracking

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/ADEN-5694

### CODE REVIEW

File: extensions/wikia/AdEngine/js/slot/service/slotRegistry.js [hunk 1]
```diff
@@ -68,10 +70,20 @@ define('ext.wikia.adEngine.slot.service.slotRegistry',  [
 		slotQueueCount = {};
 	});
 
+	function storeScrollY(slotName) {
+		scrollYOnDfpRequest[slotName] = win.scrollY || win.pageYOffset;
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/slot/service/slotRegistry.js:74]: `pageYOffset`. Do we care about IE8 and other dinosaurs?

**Review by kvas-damian — APPROVED** [review]

File: extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js [hunk 2]
```diff
@@ -73,7 +74,8 @@ define('ext.wikia.adEngine.adInfoTrackerHelper',  [
 				.replace('[', '').replace(']', '').replace(',', 'x'),
 			'viewport_height': win.innerHeight || 0,
 			'product_label': '',
-			'ad_status': status || 'unknown'
+			'ad_status': status || 'unknown',
+			'scrollY': slotRegistry.getScrollY(slot.name) || 0
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js]: https://github.com/Wikia/crossbow/pull/188/files#diff-9e1e4e645d7df09a12433c3fcf65f973R55
`scrollY` vs `scroll_y`?

**Review by kvas-damian — CHANGES_REQUESTED** [review]
Please resolve https://github.com/Wikia/app/pull/13579/files#r133885812

File: extensions/wikia/AdEngine/js/slot/service/slotRegistry.js (hunk 1 above)
Comment (dianafa) [reply extensions/wikia/AdEngine/js/slot/service/slotRegistry.js:74]: discussed offline, for ie < 9 we'd need to perform `document.documentElement.scrollLeft + document.documentElement.scrollTop` which works ONLY on those old ones IEs. We don't support IE at all: http://community.wikia.com/wiki/Help:Supported_Browsers

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**